app/javascript/src/Policies/components/PoliciesWidget.jsx

```diff
@@ -53,7 +53,7 @@ const PolicyList = ({ registry, chain, originalChain, policyConfig, ui, boundAct
     closePolicyRegistry: boundActionCreators.closePolicyRegistry
   }
 
-  const buttonsFieldset = document.querySelector('fieldset.buttons')
+  const buttonsFieldset = document.querySelector('#edit_proxy_2 > fieldset.buttons')
```

Comment:
Why #edit_proxy_2 ? What is this '2'? :thinking:

Reply:
It's the ID of the form, I guess that comes from `semantic_form_for()`.

Reply:
In APIAP, the `fieldset.buttons` is a direct child of `#edit_proxy_2`. In non-APIAP it is not. This is the way I found to affect only APIAP.

Reply:
But wouldn't `#edit_proxy_2` only be valid of a proxy with id=2 and couldn't it be any number?

Reply:
OK I didn't know that

Reply:
I'm just guessing…

Reply:
But classes would be saver than ids

Reply:
Done

Reply:
When policies are in integration edit page they sit inside a container with id `#integration-tabs` (i know, also an id). Maybe you can use that?